doc/connection-backoff.md

```diff
@@ -32,11 +32,14 @@ ConnectWithBackoff()
 
 With specific parameters of
 MIN_CONNECT_TIMEOUT = 20 seconds
-INITIAL_BACKOFF = 1 second
-MULTIPLIER = 1.6
+INITIAL_BACKOFF = 1.170820393 seconds
+MULTIPLIER = 1.618033989
 MAX_BACKOFF = 120 seconds
 JITTER = 0.2
 
+These parameters make each successive delay equal to the sum of the two previous
+delays (ignoring jitter and max backoff).
+
```

Comment:
The numbers I picked just happened to be pretty close to the existing numbers, which appear to arbitrary.  Close to an arbitrary number is still arbitrary, so it didn't seem that unpalatable.  Why not log pi or some other number?  No particular reason.  The idea was to pick a number that give a warm fuzzy feeling, and phi happens to be one of those numbers.  

Int addition is probably faster by a few nanoseconds, but since that will be used to determine how many billions of nanoseconds to wait, I don't believe it's too much of a transgression?